README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
+    * **[Proposal]** Maritime vehicle model estimation based on URDF
+      * Use vehicle mesh and parameters to estimate the Fossen equation of motion parameters
+    * **[Proposal]** Maritime vehicle estimation based experiments
+      * Use control, pose and velocity bags and URDF to estimate the Fossen equation of motion parameters
+    * **[Proposal]** Maritime vehicle model
+      * Use vehicle equations of motion to predict robot next position and velocity. This can be used with localization and control algorithms.
+  * References: 
+    *  Python lib that estimates added mass: [Capytaine](https://github.com/mancellin/capytaine)
+    *  [OpenFoam](https://www.openfoam.com/)
+
+* Guidance, Navigation and Control (GNC)  
+  * Description: Packages for gnc of maritime robots.
+  * Motive: There are some control and tasks specifics to maritime robots, such as thruster control allocation, surveilance, vessels handling of tides...
+  * Repositories
+    * **[Proposal]** Thruster manager
+  * References: 
+    *  [Plankton thruster manager](https://github.com/Liquid-ai/Plankton/blob/master/uuv_control/uuv_thruster_manager/src/uuv_thrusters/thruster_manager.py)
+ 
+* Localization
+  * Description: Packages to localize maritime vehicles.
+  * Motive: Underwater vehicles have a lot of differences regarding the sensors they can use for localizing theirselves. GPS does not works underwater, as well as lidars or electromagnetic waves for long distances... But acoustic waves work pretty well, so underwater localization rely more on sensors like DVL and Sonar, that uses acoustic waves to measure distance and velocity.  
+  * Repositories:
+    * **[Proposal]**: Multilateration of acoustic beacons
+  * References: 
+    *  [Mesh Navigation](https://github.com/uos/mesh_navigation)
+    *  [Fuse Localization](https://github.com/uos/mesh_navigation)
+    *  [MagNav](https://www.gpsworld.com/us-air-force-to-explore-navigating-with-magnetism/)
+
+* Perception
+  * Description: Packages for perception algorithms for maritime robotics.
+  * Motive: Since there are different sensors such as sonars, there is a need to process this data to obtain information. Moreover, it can be created some common structures detection.
+  * Repositories:
+    * **[Proposal]**: Common Maritime Environments Detections (buoys, valves, pipes...)
+    * **[Proposal]**: Fusion of sonar and camera data
+  * References: 
+    * Suggestions...
+   
+* Robots
+  * Description: Packages with maritime robots and structures.
+  * Motive: All the software being develop shall have not only unit tests but integration tests, so robots and structures shall be developed for this purporse. These shall include vcstool for specifying packages to be used with the robot as well as ros_ign_bridge to test it on simulation.
+  * Repositories
+    * **[Proposal]** Hover type UW vehicle
+    * **[Proposal]** Torpedo shaped UW vehicle
+    * **[Proposal]** USV
+    * **[Proposal]** Intervention UW vehicle 
```

Comment:
My reading of this is that this aims to provide control-level software for each of these different control schemas. It seems like there's an opportunity here to more effectively leverage existing ROS code: rather than providing each of these as repositories it may make sense to try to implement these in the existing [ros_control repo](https://github.com/ros-controls/ros_controllers) using the same intefaces for mobile platforms, which take in command velocities and provide odometry.

Although the typical usage for these is 2 dimensional, the interfaces themselves do support 3 dimensions. This also would cover [mabelzhang's question](https://github.com/ros-maritime/community/pull/1/files#r804976479) about "should this also cover surface-based robots": all the boats I can think of could be described as being controlled with either differential, ackerman, or holonomic controllers, all of which are already implemented.